sdk/purview/azure-purview-administration/tests/_util.py

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from azure_devtools.scenario_tests import RecordingProcessor
+import json
+
+
+class PurviewAccountRecordingProcessor(RecordingProcessor):
+    def process_response(self, response):
+        response["body"]["string"] = '{"atlasKafkaPrimaryEndpoint":"000","atlasKafkaSecondaryEndpoint":"000"}'
+        return response
+
+
+class PurviewAccountCollectionsRecordingProcessor(RecordingProcessor):
+    def process_response(self, response):
```

Comment:
@iscai-msft we want to scrub some sensitive info to avoid any potential problems. Is there any better way to do that? I don't know how to scrub it in recording files except for self-defined `RecordingProcessor` function